spec/views/user_index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'User index page', type: :feature do
+  before :each do
+    @user1 = User.create(name: 'Tomas', photo: 'https://usp.com/pto.png', bio: 'Web developer')
+    visit root_path
+  end
+
+  it 'displays username of all other users' do
+    expect(page).to have_content(@user1.name)
+  end
+
+  it 'displays profile picture for each user' do
+    expect(page).to have_css('img')
```

Comment:
- [ ] A quick fix following the example I gave earlier, `expect(page).to have_content(all_user.profile_pic)`

Reply:
@addod19, same as with the response from above.